src/qt/bitcoinstrings.cpp

```diff
@@ -194,7 +194,7 @@ QT_TRANSLATE_NOOP("bitcoin-core", "Connect only to the specified node(s)"),
 QT_TRANSLATE_NOOP("bitcoin-core", "Connect through SOCKS5 proxy"),
 QT_TRANSLATE_NOOP("bitcoin-core", "Connect to a node to retrieve peer addresses, and disconnect"),
 QT_TRANSLATE_NOOP("bitcoin-core", "Connection options:"),
-QT_TRANSLATE_NOOP("bitcoin-core", "Copyright (C) 2009-%i The Bitcoin Core Developers"),
+QT_TRANSLATE_NOOP("bitcoin-core", "Copyright (C) 2009-%i The Zcash Developers"),
```

Comment:
Bitcoin Core copyright notices can't be deleted. In this case it wouldn't matter very much because we're about to remove the QT wallet anyway, but can you remove either this change, or all the QT changes?